sqlite/db.go

```diff
@@ -0,0 +1,46 @@
+package sqlite
+
+import (
+	"database/sql"
+	"fmt"
+
+	_ "github.com/mutecomm/go-sqlcipher" // We require go sqlcipher that overrides default implementation
+)
+
+// The reduced number of kdf iterations (for performance reasons) which is
+// currently used for derivation of the database key
+// https://github.com/status-im/status-go/pull/1343
+// https://notes.status.im/i8Y_l7ccTiOYq09HVgoFwA
+const kdfIterationsNumber = 3200
+
+func openDB(path string, key string, kdfIter int) (*sql.DB, error) {
+	db, err := sql.Open("sqlite3", path)
+	if err != nil {
+		return nil, err
+	}
+
+	keyString := fmt.Sprintf("PRAGMA key = '%s'", key)
```

Comment:
hmm, that looks like a perfect place for an SQL injection... is there any safer way to execute it? like maybe `db.Exec("PRAGMA key = ?", key)` or something like that

Reply:
i don't think that you can use arguments in pragma statements. somewhat hard to trace the whole flow due to c code use in the library. but i wasn't able to use ? or $1 in a statement

Reply:
can we hash the key or make a hex-representation to make sure that we don't have any unsafe characters here? cc @cammellos

Reply:
i don't think that i will end up using this storage, but we need to fix it in the original place if there are some concerns

Reply:
what does "the original place" means? the PFS db?

Reply:
yes, PFS db